TeadsSampleApp/Controllers/InRead/Direct/CollectionView/InReadDirectCollectionViewController.swift

```diff
@@ -11,21 +11,30 @@ import TeadsSDK
 
 class InReadDirectCollectionViewController: TeadsViewController {
     
+    enum TeadsElement: Equatable {
+        case article
+        case ad(_ ad: TeadsInReadAd)
+        case trackerView(_ trackerView: TeadsAdOpportunityTrackerView)
+    }
+    
     @IBOutlet weak var collectionView: UICollectionView!
     
     let contentCell = "TeadsContentCell"
     let teadsAdCellIndentifier = "TeadsAdCell"
     let fakeArticleCell = "fakeArticleCell"
-    let adItemNumber = 2
+    let trackerViewItemNumber = 2
+    var adItemNumber: Int {
+        return trackerViewItemNumber + 1
+    }
```

Comment:
I understand this part but it seems a little bit complicated to understand. Maybe be adding some comments to know what is the purpose of this two variables could be helpful?